bumble/l2cap.py

```diff
@@ -1865,11 +1867,45 @@ def on_l2cap_connection_parameter_update_request(
                 ),
             )
 
+    async def update_connection_parameters(
```

Comment:
I think it would be good to also expose this functionality via the existing `Device.update_connection_parameters` (and indirectly via `Connection.update_parameters`). Maybe adding a `use_l2cap` bool parameters, with default to `False` (and checking that we're in the peripheral role when that parameter is `True`).
We can keep this function here and delegate to it from the `Device` API.

Reply:
Done. Please let me know if there is room for improvement